src/App.js

```diff
@@ -9,7 +10,7 @@ function App() {
   const [results, setResults] = useState(null);
 
   useEffect(() => {
-    fetch("https://api.openweathermap.org/data/2.5/weather?q=" + city + "&units=metric" + "&appid=" + process.env.REACT_APP_APIKEY)
+    fetch("https://api.openweathermap.org/data/2.5/weather?q=" + city + "&units=metric&appid=" + process.env.REACT_APP_APIKEY)
```

Comment:
Code improvement Suggestion: Please use template literals (a.k.a backticks) for string concatenation.

Reply:
I have updated with template literals for string concatenation.